lib/cjs/generated/rum.d.ts

```diff
@@ -1755,6 +1755,10 @@ export interface StreamSchema {
          * how much did the media progress since the last context update (in ms)
          */
         watch_time?: number;
+        /**
+         * Percentage of amount of time watched relative to its total duration
```

Comment:
Suggestion: To make the description a bit clearer.
```suggestion
         * Percentage of the media watched relative to its total duration
```